tests/filecheck/dialects/vector/vector_ops.xdsl

```diff
@@ -18,7 +18,7 @@ func.func() ["sym_name" = "vector_test", "function_type" = !fun<[!memref<[4 : !i
 
 // CHECK: builtin.module() {
 // CHECK-NEXT: func.func() ["sym_name" = "vector_test", "function_type" = !fun<[!memref<[4 : !index, 4 : !index], !index>, !vector<[true], !i1>, !index], []>, "sym_visibility" = "private"] {
-// CHECK-NEXT: ^{{.*}}(%{{.*}} : !memref<[4 : !index, 4 : !index], !index>, %{{.*}} : !index):
+// CHECK-NEXT: ^{{.*}}(%{{.*}} : !memref<[4 : !index, 4 : !index], !index>, %{{.*}} : !vector<[true], !i1>, %{{.*}} : !index):
```

Comment:
I am confused. Why did this change?

Reply:
The [branch](https://github.com/xdslproject/xdsl/blob/d670605e166021e632299fceb08583ef9c66f41b/tests/filecheck/dialects/vector/vector_ops.xdsl#L6) gets that vector as one of its arguments. So filecheck expected IR should also contain it as an argument.

Reply:
Yes, but that means your initial test was wrong at this line and that did not trigger the test to fail. That's mainly what surprises me here. I guess the {{.*}} "consumed" the stuff that was printed too much... 
This is really not a very good way of testing...

Reply:
I agree. I tried catching this error with [`--match-full-lines`](https://filecheck.readthedocs.io/en/stable/06-options.html#match-full-lines) flag, but even then filecheck couldn't detect it on its own

Reply:
AFAIK, with the actual llvm filecheck, one could solve this. There you can give strings a tag and then check that the same string occurs in other positions. However, that's a feature that didnt make it into the python version yet...

Reply:
So what about this?

Reply:
I created [an issue](https://github.com/xdslproject/xdsl/issues/428) to keep track of this behaviour once this patch lands.

Reply:
We can merge. This issue is mich bigger than this PR and will need to be adressed in a separate one,IMO.